### PR TITLE
Restored Emulator For WPI 2020 Updates

### PR DESCRIPTION
**Pull request by matthew17754**
Restores the HEL emulator for performing service updates for compatibility with WPIlib 2020 for the January release. 
This restores the emulator as-is from my fork to the latest pre-modernization commit [08d444b](https://github.com/Autodesk/synthesis/commit/08d444b267fefcf5c93d3e590866dda2cae754eb).

### CODE REVIEW

**Review by NickBurrell — APPROVED** [review]
Looks like all the files are there. Looks good to me.

**Review by copernicium — APPROVED** [review]
Looks like everything's there to me.

File: emulation/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# FRC Robot Code Emulation
+
+## Overview
+The emulator is the Synthesis tool designed to help users test their FRC robot code ([Getting started with FRC control system](https://wpilib.screenstepslive.com/s/currentCS "FRC Control System")). Users can upload their own user code as the FRC user programs they would normally deploy to the RoboRIO. Its normal communication with hardware is then redirected to the [engine](../engine "Engine Source") for simulation and testing.
```
Comment (Daltz333) [inline emulation/README.md:4]:
```suggestion
The emulator is the Synthesis tool designed to help users test their FRC robot code ([Getting started with FRC control system](https://docs.wpilib.org "FRC Control System")). Users can upload their own user code as the FRC user programs they would normally deploy to the RoboRIO. Its normal communication with hardware is then redirected to the [engine](../engine "Engine Source") for simulation and testing.
```

Comment (matthew17754) [reply emulation/README.md]: Implemented. Thanks for the suggestion. Guess they updated the docs link.

**Pull request merged**